### DISCUSSION TITLE
First label of start axis(0, 0) is not shown when VerticalLabelPosition.Top and HorizontalLabelPosition.Inside

### DISCUSSION THREAD

**Discussion by ThomasCapelli** [patrykandpatrick/vico#977] [Questions]
### Question

First label of start axis(0, 0) is not shown when VerticalLabelPosition.Top and HorizontalLabelPosition.Inside, I have enough space due to extra padding that I have added to my graph but the label is still hidden. How do I manage to show it?

### Vico version(s)

2.0.0-beta.3

### UI framework(s)

Jetpack Compose

**Comment by Gowsky** [comment]
Hello, @ThomasCapelli. This is a bug, and it’s been reported as such in #1090. Please stay tuned there.
